specification/metrics/effectivecost.md

```diff
@@ -19,24 +19,25 @@ Effective Cost
 
 ## Description
 
-The cost inclusive of amortized upfront fees, amortized recurring fees, and the usage cost of the line item.
+A charge inclusive of the impacts of all reduced rates and discounts, augmented with the amortization of relevant purchases (one-time or recurring) paid to cover future eligible charges.
 
 ### Concerning Granularity and Distribution of Recurring Fee
 
 Providers should distribute the commitment purchase amount instead of including a line item at the beginning of a period so practitioners do not need to manually distribute the fee themselves.
 
 ### Concerning Amortization Approaches
 
-The amortization of the upfront charge should be amortized using a methodology determined by the provider that reflects the needs of their customer base and is proportional with usage quantity and the time granularity of the line item.  Should a practitioner desire to amortize upfront fees using a different approach, the practitioner can do so using the Billed Cost for the line item representing the initial purchase.
+The amortization of the purchase should be amortized using a methodology determined by the provider that reflects the needs of their customer base and is proportional with the Quantity In Pricing Unit and the time granularity of the line item. Should a practitioner desire to amortize relevant purchases using a different approach, the practitioner can do so using the [Billed Cost](#billedcost) for the line item representing the initial purchase.
```

Comment:
Is this true? Not all purchases include upfront fees.

Reply:
Is it eligible purchases?

Reply:
Changed to: _**Eligible purchases** should be amortized using a methodology determined by the provider that_

Reply:
LGTM